src/App.js

```diff
@@ -21,7 +26,7 @@ const saveState = () => {
 };
 const GA_TRACKING_ID = 'UA-124331187-2';
 
-store.subscribe(function(){console.log(store.getState())});
+// store.subscribe(function(){console.log(store.getState())});
```

Comment:
Can we restore this?  It is very useful for debugging.

Reply:
Sure.